### PR TITLE
Symfony 4 compatibility

### PR DESCRIPTION
**Pull request by ThomasLandauer**
As promised in https://github.com/Payum/Payum/issues/787#issuecomment-448248663 :-)

I'm not done yet...

### CODE REVIEW

File: docs/symfony/get-it-started.md [hunk 1]
```diff
@@ -71,65 +73,78 @@ class PaymentToken extends Token
 
 ```php
 <?php
-namespace Acme\PaymentBundle\Entity;
+// src/EntityPayment.php
```
Comment (makasim) [inline docs/symfony/get-it-started.md]: should be `src/Entity/Payment.php`

File: docs/symfony/get-it-started.md [hunk 2]
```diff
@@ -71,65 +73,78 @@ class PaymentToken extends Token
...
     gateways:
         offline:
             factory: offline
 ```
 
-_**Note**: You can add other gateways to the gateways section too._
+For Symfony >=4, you need to add this to your `config/services.yaml`:
+```yml
+services:
+    Payum\Core\Payum:
```
Comment (makasim) [inline docs/symfony/get-it-started.md]: this could be added to the PayumBundle.

File: docs/symfony/get-it-started.md (hunk 1 above)
Comment (ThomasLandauer) [reply docs/symfony/get-it-started.md]: OK, see next commit

File: docs/symfony/get-it-started.md (hunk 2 above)
Comment (ThomasLandauer) [reply docs/symfony/get-it-started.md]: Cool, could you do that?

**Comment by ThomasLandauer** [comment]
Please see https://github.com/Payum/Payum/issues/789 first, I might need to re-delete something again...

File: docs/symfony/get-it-started.md (hunk 2 above)
Comment (ThomasLandauer) [reply docs/symfony/get-it-started.md]: Done :-) See https://github.com/Payum/PayumBundle/pull/482

**Comment by tristanbes** [comment]
Thanks for your work, we can't wait until it gets merged to use Payum with Symfony4+

**Comment by tristanbes** [comment]
ping :)

**Comment by ThomasLandauer** [comment]
Sorry, I'm not using payum anymore.

**Comment by petarvasilev91** [comment]
I am still getting the error after adding "Payum\Core\Payum:" to the "services.yaml"

File: docs/symfony/get-it-started.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ _**Note**: Where payum/offline is a php payum extension, you can for example cha
 
 _**Note**: Use payum/payum if you want to install all gateways at once._
 
-Enable the bundle in the kernel:
+Enable the bundle in the kernel (only necessary for Symfony <=3):
```
Comment (pierredup) [inline docs/symfony/get-it-started.md:25]: Symfony 3 isn't supported anymore in PayumBundle, so I think all references and config for Symfony 3 can be removed

File: docs/symfony/get-it-started.md [hunk 4]
```diff
@@ -37,99 +37,125 @@ public function registerBundles()
...
-        Acme\PaymentBundle\Entity\Payment: { doctrine: orm }
+        App\Entity\Payment: { doctrine: orm }
             
     gateways:
         offline:
             factory: offline
 ```
 
-_**Note**: You can add other gateways to the gateways section too._
+To enable autowiring, you need to add this to your `config/services.yaml`:
```
Comment (pierredup) [inline docs/symfony/get-it-started.md:122]: This part can be removed since it's handled automatically in the bundle now.

**Pull request merged**